amplify/data/resource.ts

```diff
@@ -19,6 +20,7 @@ const schema = a
         id: a.id().required(),
         firstName: a.string(),
         lastName: a.string(),
+        role: a.string(),
         email: a.string(),
```

Comment:
Let's add a default here
```suggestion
        role: a.string().default("Participant"),
```

Reply:
Ensure you update the Pre-sign up backend function to also set this field to Participant on creation since you added it as a new field